frontend/schemas/project.ts

```diff
@@ -0,0 +1,53 @@
+import { useIntl } from 'react-intl';
+
+import { object, string, array, number, mixed, boolean } from 'yup';
+
+export default (page: number) => {
+  const { formatMessage } = useIntl();
+
+  const messages = {
+    project_gallery: {
+      min_lenth: formatMessage({ defaultMessage: 'Upload at least one picture', id: 'BG1UW6' }),
```

Comment:
🟠

This field is optional so there shouldn't be any minimum number of pictures.